SecurityPkg/Tcg/Tcg2Pei/Tcg2Pei.c

```diff
@@ -644,6 +648,38 @@ MeasureCRTMVersion (
            );
 }
 
+// MU_CHANGE [BEGIN] - Measure Firmware Debugger Enabled
+
+/**
+  Measure and log firmware debugger enabled, and extend the measurement result into a specific PCR.
+
+  @retval EFI_SUCCESS           Operation completed successfully.
+  @retval EFI_OUT_OF_RESOURCES  Out of memory.
+  @retval EFI_DEVICE_ERROR      The operation was unsuccessful.
+**/
+EFI_STATUS
+MeasureFirmwareDebuggerEnabled (
+  VOID
+  )
+{
+  TCG_PCR_EVENT_HDR  TcgEvent;
+
+  TcgEvent.PCRIndex  = 7;

```

Comment:
Are we still planning on also measuring this to PCR 0?

Reply:
In a separate commit not to be upstreamed, I think yes. @cfernald, right, we wanted to use PCR 0 to indicate the code is in an unknowable state?

Reply:
Offline we determined if we want to measure into PCR 0, it should be in a separate PR